### PR TITLE
LPS-167867 Created the merge subtask action

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx [hunk 1]
```diff
@@ -347,14 +380,19 @@ const TestFlowTasks = () => {
...
-						filter: searchUtil.eq('taskId', testrayTask.id),
+						filter: subTaskFilter,
 					}}
 				>
-					{({items}, {dispatch, listViewContext: {selectedRows}}) => {
+					{(
+						{items},
+						{dispatch, listViewContext: {selectedRows}, mutate}
+					) => {
+						const dataFromSelectedRows: TestraySubTask[] = selectedRows.map(
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx:390]:
```suggestion
						const selectedSubtasks: TestraySubTask[] = selectedRows.map(
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx [hunk 2]
```diff
@@ -347,14 +380,19 @@ const TestFlowTasks = () => {
...
+							(rowId) => items.find(({id}) => rowId === id)
+						);
+
 						const alerts = getFloatingBoxAlerts(
-							selectedRows.map((rowId) =>
-								items.find(({id}) => rowId === id)
-							)
+							dataFromSelectedRows
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx:395]:
```suggestion
							selectedSubtasks
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Good job!

See my suggestions and add this property in QaTable.tsx

image

**Comment by rafaelfs98** [comment]
@kevenleone  resolved sugestions

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**